index.js

```diff
@@ -2,6 +2,34 @@ const divbooks = document.querySelector('.books');
 const inputTitle = document.querySelector('#title');
 const inputAuthor = document.querySelector('#author');
 const addBtn = document.querySelector('#add');
+const listNav = document.querySelector('#list_nav');
+const addNav = document.querySelector('#add_nav');
+const contactNav = document.querySelector('#contact_nav');
+const listSection = document.querySelector('#all_books');
+const addSection = document.querySelector('#add_book');
+const contactSection = document.querySelector('#contact_info');
+const date = document.querySelector('.date');
+const lux = window.luxon;
+const { DateTime } = lux;
+const localTime = DateTime.local();
+date.innerHTML = DateTime.fromISO(localTime).toLocaleString(DateTime.DATETIME_MED);
+contactNav.addEventListener('click', () => {
+  contactSection.style.display = 'block';
+  listSection.style.display = 'none';
+  addSection.style.display = 'none';
+});
+
+addNav.addEventListener('click', () => {
+  addSection.style.display = 'block';
+  listSection.style.display = 'none';
+  contactSection.style.display = 'none';
+});
+
```

Comment:
- [ ] Good Job guys, however, let us fix this or work on something to get approved while adding a book, let us make sure that the current data are not kept in the input fields once a user clicks on the add button so that you can add another data later.

![date-2](https://user-images.githubusercontent.com/55635977/138284485-43f006e1-ecc9-4e60-8f70-387b89f71de8.png)